### PR TITLE
Install file update

### PR DESCRIPTION
**Pull request by ar13pit**
Cleanup and addition of commands to make all dependencies for the speech_recognition repo

### CODE REVIEW

**Comment by ar13pit** [comment]
Do not merge without testing.

File: install.bash [hunk 1]
```diff
@@ -40,13 +40,13 @@ sudo apt-get install --assume-yes build-essential git python python-scipy sox sw
 # Kaldi install dependencies
 _kaldi_install_dependencies()
 {
+    echo "Checking and installing dependencies..."
     # Change exit to return to source check_dependencies and change back once done
     sed -i "s|exit|return|g" $KALDI/tools/extras/check_dependencies.sh
-    source $KALDI/tools/extras/check_dependencies.sh
+    source $KALDI/tools/extras/check_dependencies.sh > /dev/null 2>&1
```
Comment (dvdinth) [inline install.bash]: Why would you do `> /dev/null 2>&1` here? I could see why you would do simply `> /dev/null` to suppress the STDOUT, but why would you also want to mute the STDERR here?

File: install.bash [hunk 2]
```diff
@@ -56,6 +56,14 @@ _kaldi_build()
 {
     _kaldi_install_dependencies
 
+    # Check g++ version before starting build
+    gpp_version_num=$(g++ --version | grep ^g++ | sed 's/^.* //g' | sed 's/\./ /g' | xargs printf "%d%02d%02d")
+    if [ $gpp_version_num -gt 70000 ]
```
Comment (dvdinth) [inline install.bash:75]: This doesn't work on my machine. When I run:
`g++ --version | grep ^g++`
I get:
`g++ (Ubuntu 5.4.0-6ubuntu1~16.04.11) 5.4.0 20160609`

Hence for me `gpp_version_num = 201606090000`, which is definitely greater than 7000, but I don't have version 7.0 (I have 5.4.0 as can be seen from the line above)

**Comment by dvdinth** [comment]
Haven't tested this entirely, but other than above comments it seems OK (haven't tested the Sequitur commit)

File: install.bash (hunk 2 above)
Comment (ar13pit) [reply install.bash:75]: Yeah I noticed that when I was testing. Working on a fix.

File: install.bash (hunk 1 above)
Comment (ar13pit) [reply install.bash]: Valid point. Will make a change there.

**Comment by ar13pit** [comment]
@DvanDinther @MatthijsBurgh review needed. Merge only after test.

**Comment by dvdinth** [comment]
Seems good, didn't test

**Review by dvdinth — APPROVED** [review]

**Pull request merged**